source/beamline/CLS/CLSBeamlineStatus.cpp

```diff
@@ -40,6 +40,14 @@ bool CLSBeamlineStatus::isOff() const
 	}
 }
 
+AMControl * CLSBeamlineStatus::beamlineStatusPVControl()
+{
+	if (beamlineStatusPVControl_ )
+		return beamlineStatusPVControl_ ;
+	else
+		return this;
```

Comment:
But that breaks the paradigm used everywhere else in the framework.  To be consistent, there should be a flag like `hasExternalStatusControl()` or something similar and then you know to ask for the control explicitly through `beamlineStatusControl()` or use the control itself.

The change by itself isn't really a big deal, but it's context in relation to the rest of the code is important.